input/pagecontent/RAD-Y1.md

```diff
@@ -68,11 +61,56 @@ This URL is configurable by the Responder and is subject to the following constr
+
+The FHIR Bundle.meta.profile shall have the following value: `http://profiles.ihe.net/RAD/IMR/StructureDefinition/imr-bundle`
+
+The [IMR Bundle](StructureDefinition-imr-bundle.html): 
+  - shall be a [Transaction Bundle](http://hl7.org/fhir/http.html#transaction)
+  - shall create one [IMR DiagnosticReport](StructureDefinition-imr-diagnosticreport.html)
+  - may create/update/read one or more [IMR ServiceRequest](StructureDefinition-imr-servicerequest.html)
+  - may create/update/read one [Patient](http://hl7.org/fhir/patient.html)
+  - may create/read one or more [Organization](http://hl7.org/fhir/organization.html)
+  - may create/read one or more [Practitioner](http://hl7.org/fhir/practitioner.html)
+  - may create/read one or more [IMR Observation](StructureDefinition-imr-observation.html)
+  - may create/read one or more [IMR ImagingStudy](StructureDefinition-imr-imagingstudy.html)
+
+The Sender may set meta.profile of each resource to be the corresponding IMR profile. This enables the recipients to validate the received resource against the IMR resource profile specification.
+
+> Note that a Sender may choose not to set meta.profile to a specific profile, or may set it to multiple profiles.
+
+The Sender shall create corresponding properly identifiable resources unless the proper record keys or absolute identification information is not available. Identifiable resources are preferred because each resource in the bundle is valuable as a standalone resource outside the context of the DiagnosticReport Resource (e.g. independently searchable, and the same resource can be referenced multiple times).
+
+When resources are `contained`, they shall be contained using the FHIR contained method (see [http://hl7.org/fhir/references.html#contained](http://hl7.org/fhir/references.html#contained) ).
+
+The Sender shall encode the rendered report that is referenced by DiagnosticReport.presentedForm in one of the following ways:
+- Encode the rendered report as a base64Binary in DiagnosticReport.presentedForm.data
+- Encode the rendered report as a base64Binary in a [Binary Resource](https://www.hl7.org/fhir/binary.html) and this Binary Resource is referenced in DiagnosticReport.presentedForm.url.
+  - The Binary Resource shall be in the Bundle. See FHIR Resolving references in Bundles at [http://hl7.org/fhir/bundle.html#references](http://hl7.org/fhir/bundle.html#references). 
+- Host the rendered report somewhere and provide the url in DiagnosticReport.presentedForm.url.
+
+The Sender shall populate accurate .hash and .size for the rendered report content in presentedForm: 
+* Where the presentedForm is a Binary Resource instance, the .hash and .size measure the raw artifact that has been base64encoded in the Binary.data element.  
+* Where the presentedForm is hosted elsewhere, not as a Binary Resource, the .hash and the .size shall represent the rendered report content that would be retrieved using the mime-type specified in contentType element. 
```

Comment:
Should this block of text be part of 2.4.Y1.4.1.2.2.2 together with all the other requirements related to rendered report?

Reply:
It is in 2.4.Y1.4.1.2.2.2 in my RAD-Y1 branch.   Do you disagree with putting it there?

should I move it?